crowdin_api/api_resources/bundles/resource.py

```diff
@@ -127,6 +132,68 @@ def edit_bundle(self, projectId: int, bundleId: int, data: Iterable[BundlePatchR
             request_data=data,
         )
 
+    def download_bundle(
+        self,
+        projectId: int,
+        bundleId: int,
+        exportId: str
+    ):
+        """
+        Download bundle.
+
+        Link to documentation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.bundles.exports.download.get
+
+        Link to documentation for enterprise:
+        https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.bundles.exports.download.get
+        """
+
+        return self.requester.request(
+            method="get",
+            path=f"{self.get_bundles_exports_path(projectId=projectId, bundleId=bundleId, exportId=exportId)}/download",
+        )
+
+    def export_bundle(
+        self,
+        projectId: int,
+        bundleId: int
+    ):
+        """
+        Export bundle.
+
+        Link to documentation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.bundles.exports.post
+
+        Link to documentation for enterprise:
+        https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.bundles.exports.post
+        """
+
+        return self.requester.request(
+            method="post",
+            path=f"{self.get_bundles_exports_path(projectId=projectId, bundleId=bundleId)}",
```

Comment:
You don't need to use the `f` string format if the method return string. Could you please remove it?